shared/src/persistence/elasticsearch/getCasesClosedByJudge.ts

```diff
@@ -13,16 +17,33 @@ import { search } from './searchClient';
 export const getCasesClosedByJudge = async ({
   applicationContext,
   endDate,
-  judgeName,
+  judges,
   startDate,
-}) => {
+}): Promise<CasesClosedType> => {
   const source = ['status'];
 
-  const { results } = await search({
+  const shouldFilters: QueryDslQueryContainer[] = [];
+
+  if (judges.length) {
+    judges.forEach(judge => {
```

Comment:
For this report, I believe you can only choose one judge to filter by or display all judges, why not just pass a specific judge ID if the user has selected to run the report for just one judge? I think that would simplify a lot of the querying code.

Reply:
From my understanding, if we don't specify the judges to make the queries for, the results will return the cases closed for all judges in the db (including legacy judges).